packages/node/edge/index.ts

```diff
@@ -0,0 +1,36 @@
+import type { LogtoConfig, ClientAdapter } from '@logto/client';
+import { createRequester } from '@logto/client';
+
+import BaseClient from '../src/client.js';
+
+import { generateCodeChallenge, generateCodeVerifier, generateState } from './generators.js';
+
+export default class LogtoClient extends BaseClient {
```

Comment:
should we add some comments to say this client is for NextJS edge runtime? also since it's exclusively for NextJS (if i understand correctly), shall we just put this into `@logto/next`?

Reply:
This is not exclusively for NextJS, in the future we may support other edge runtime (Cloudflare workers). Comments added.

Reply:
cool

Reply:
did you forget to push? cannot find the comments

Reply:
updated